docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -227,70 +226,43 @@ Prefer a video? Expand for our installation video using PolkadotJS interface.
 
 ![NStaking-8](/img/doc-imgs/operators-staking/NStaking-8.png)
 
 :::info
-It can take up to 10 minutes for the operator to be registered and appear on the page. 
-You can check if the operator was created successfully by following the [steps](#checking-your-operatorid).
-:::
-
-:::tip
-You can view some additional actions by clicking on `action` next to your operator. 
-You can increase your stake, withdraw some stake and de-register your operator from there. 
+Make sure to either locate the signing key file generated on the previous **[Create operator key](#create-operator-key)** step or get use the key phrase that was displayed when the key was created and is also in that key file.
 :::
 
-If you prefer to use a native PolkadotJS, or faced any issues in one of the steps above, you can try using [PolkadotJS interface guide](tips-operator#register-an-operator-using-polkadotjs-interface-alternative-way) to register an operator on the domain. 
-
-### Checking your operatorId
-
-There are two ways to check your operatorId: 
-
-1. You can use PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
-
-![Staking-7](/img/doc-imgs/operators-staking/Staking-7.png)
-2. Browse the **recent events** and you should see **domains.OperatorRegistered** event.
-
-![Staking-8](/img/doc-imgs/operators-staking/Staking-8.png)
-3. Click on the dropdown arrow to view the **domainId** and **operatorId**.
-
----
-
-Alternatively, you can use [Subscan](https://subspace.subscan.io/) which is a little easier to navigate for this job. 
-1. Navigate to **[Subspace Subscan](https://subspace.subscan.io/)** portal.
-2. Click on `Blockchain` -> `Extrinsics`.
-
-![Staking-9](/img/doc-imgs/operators-staking/Staking-9.png)
-
-3. Scroll to the bottom of the page to view all recent events, search for `register_operator` event.
-
-![Staking-10](/img/doc-imgs/operators-staking/Staking-10.png)
-
-4. Click on `Extrinsic ID` for the desired event.
+7. Approve the request in the pop-up window. 
 
-5. Scroll to `Parameters` and ensure that `signing_key` corresponds to your signing key.
+![NStaking-9](/img/doc-imgs/operators-staking/NStaking-9.png)
 
-![Staking-11](/img/doc-imgs/operators-staking/Staking-11.png)
+8. Congratulations, you're now registered as an **operator**!  You can click on the Manage My Operator tab to see your operator and get the Operator Id.
 
-6. Scroll to `Events` and click on dropdown arrow for `domains(OperatorRegistered)`.
+![NStaking-10](/img/doc-imgs/operators-staking/NStaking-10.png)
 
-![Staking-12](/img/doc-imgs/operators-staking/Staking-12.png)
+:::info
+It can take up to 10 minutes for the operator to be registered and appear on the page.  It may take a day before your Stake amount is displayed under Total Stake in the grid.  Once you have your Operator Id you can restart your operator node and include the `--operator-id **OPERATOR_ID**` as a parameter. 
```

Comment:
Is this 10 minutes for the squids to index? And I'm not sure where the day comes from here - it should just be an epoch transition I think.

Reply:
It has been about 18 hours since I registered my test operator and it still shows 0 tSSC.  That is why I assumed it would take a day.